README.md

```diff
@@ -73,6 +73,15 @@ The following options are also supported and will be passed directly to the
 - [`lastModified`](https://www.npmjs.com/package/send#lastmodified)
 - [`maxAge`](https://www.npmjs.com/package/send#maxage)
 
+#### `wildcard`
+
+Default: `true`
+
+If set to `true`, `fastify-static` adds a wildcard route to serve files.
+If set to `false`, `fastify-static` globs the filesystem for all defined
+files in the served folder, and just creates the routes needed for
+those.
+
```

Comment:
"The default options of https://www.npmjs.com/package/glob are applied for getting the file list"?